bazel/foreign_cc/io_opencensus_cpp.patch

```diff
@@ -0,0 +1,52 @@
+--- copts.bzl	2019-06-03 16:42:49.752903373 -0700
```

Comment:
What is the long term plan for this?

Reply:
When envoy upgrades to a newer absl, we can drop io_opencensus_cpp patch.

@kyessenov is working on this right now in https://github.com/envoyproxy/envoy/pull/7146 (thanks!)